src/ResourceDetailView/BasicTextCard.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <!-- 
+    VCard comes from Vuetify: https://vuetifyjs.com/en/components/cards/
+    and provide a great way to wrap related content together.
+    The "tonal" variant is a built-in customization to change the overall card appearance
+  -->
+  <VCard
+    :title="info.label"
+    class="mb-2"
+    variant="tonal"
+  >
+  <!-- 
+  VCardText slot sits below the header and can be considered the main body of the card
+  -->
+    <VCardText>
+    <!-- Double brackets to simply render the `info.value` -->
+    {{ info.value}}
+    </VCardText>
+  </VCard>
+</template>
+
+
+<script setup>
+/**
+ * Unlike some of the other example components which focus on specific use cases, 
+ * sometimes you will want to create generic components with reusability
+ * As long as this component recieves an `info` object that has a `label` and `value` properties
+ * it will render a consistent VCard result that can be used anywhere
+ */
+
+/**
+ * @typedef {object} Props
+ * @property {object} Props.info - The simple info object provided to the component
+/** @type {Props} */
+
+defineProps({
+  info: {
+    type: Object,
+    required: true,
+  },
+});
```

Comment:
Two changes I'd love to request here to simplify / improve the reusability here (feel free to debate) to model some best practices (imho):
 - instead of a single `info` prop that just includes two sub-props, have each piece of independent information be a separate prop - a string `title` prop and a string `info` prop. This helps each prop be better defined and clear in what it expects.
 - Since `value` prop just renders as a string in the template and we don't do anything else with it, I'd suggest it should be the default slot instead. That'd have two benefits:
   - This component could be re-used for any nested info
   - We can show an example of slots

Reply:
Since info isn't anything but a carrier of the `value` and `title` I've swapped to just a title prop and slot for the default content

Reply:
Thanks!